### ISSUE TITLE
Weird mystical mechanics issue only with current 1.5 Hotfix and Forge 14.23.4.2758

### ISSUE THREAD

**Issue by Dragnier** [DaedalusGame/EmbersRekindled#43]
When using Forge 14.23.4.2757, the current 1.5 Hotfix version works just fine. However, updating to Forge 14.23.4.2758 causes the Mystical Mechanics dependency issue again. [Link to Gist of the Crash Report](https://gist.github.com/Dragnier/0b630adeafaba9211211e6921f258177). All of my mods and configs are the same; the only difference is the Forge version.

Edit: I just noticed the crash report shows the Forge version incorrectly compared to what Curse shows when this happens... It still shows 2757 even though Curse shows 2758.

**Comment by DaedalusGame** [comment]
your crashlog says you're using 1.5, not 1.5hotfix

**Comment by Dragnier** [comment]
Curse says otherwise. It says 1.5hotfix... just like the messed up Forge version. Maybe it's actually something wrong with Forge 2758 or Curse itself?

**Comment by DaedalusGame** [comment]
i think this may be a curse problem. check your mods folder for what you're actually running. the crashlog never lies about this kinda stuff and apparently curse has been shipping corrupted forge versions recently so i wouldn't be surprised in the slightest

**Comment by Dragnier** [comment]
Ok, I made sure the versions were correct in the folders. Then I deleted the 2758 Forge version to force  Curse to re-download it, just in case. Now it just crashes when I hit play before even loading and gives no crash report... so I think it must either be that Forge version or Curse corrupting it. As a result, I'll close this here, since it isn't your mod causing the issue.

I'm also going to download Forge 2758 directly and replace the file manually to see which it is so I can report it.

Edit: I can officially say it is Curse corrupted as the manual download and install of 2758 works just fine. Off to their support page...
